### PR TITLE
Add guard-livereload for automatic refresh

### LINKED ISSUE #137: Configure hot reload for views
During the redesign effort, we noticed it'd be nice to have an automatic reload for view changes.
Suggestion: https://github.com/guard/guard-livereload

### PR DESCRIPTION
**Pull request by ramos-ph**
This PR adds a live reload feature to improve the development experience.

Closes #137

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ A simple electronic punch clock to track hours spent on projects.
 Ruby 2.7.5
 Rails 6.0.3
 Postgres >= 9.1
+Node 16.13.1
```
Comment (ramos-ph) [inline README.md:15]: Had some issues with installing.
Although not related to the PR, I think it will be better to specify the Node version.

**Review by wenderjean — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (adbatista) [reply README.md:15]: nice, the package.json should already handle this but is nice to also have here, just need to pay attention to keep updated

**Comment by adbatista** [comment]
specs breaking, maybe rebasing will fix it

**Review by adbatista — APPROVED** [review]

**Pull request merged**